docs/src/topics/flavors/rke2.md

```diff
@@ -0,0 +1,35 @@
+# RKE2
+## Specification
+| Control Plane               | CNI    | Default OS   | Installs ClusterClass |
+|-----------------------------|--------|--------------|-----------------------|
+| [rke2](https://docs.rke2.io/) | Cilium | Ubuntu 22.04 | No                    |
+## Prerequisites
+* [Quickstart](../topics/getting-started.md) completed
+* Select an [rke2 kubernetes version](https://github.com/rancher/rke2/releases) to set for the kubernetes version
+  ```bash
+  export KUBERNETES_VERSION=v1.29.1+rke2r1
+  ```
+* Installed [rke2 bootstrap provider](https://github.com/rancher-sandbox/cluster-api-provider-rke2) into your management cluster
+  * Add the following to `~/.cluster-api/clusterctl.yaml` for the k3s bootstrap/control plane providers
```

Comment:
it looks like the rke2 provider info is built into clusterctl, so I don't think this is necessary